x-pack/plugins/monitoring/common/http_api/beats/post_beat_detail.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as rt from 'io-ts';
+import { ccsRT, clusterUuidRT, timeRangeRT } from '../shared';
+
+export const postBeatDetailRequestParamsRT = rt.type({
+  clusterUuid: clusterUuidRT,
+  beatUuid: rt.string,
```

Comment:
It only appears once so it's not really important, but do you think it's worth adding `beatUuid` to `shared` so it's a proper "entity" like `clusterUuid`?

Reply:
My first intuition was to do just that, but then I saw the singular usage and decided against it. :shrug: Do you mind if we revisit this once all the APIs are converted?